src/js/containers/ValidationProgress.jsx

```diff
@@ -0,0 +1,20 @@
+import { connect } from 'react-redux'
+import ValidationProgress from '../components/ValidationProgress.jsx'
+
+function mapStateToProps(state) {
+  const {
+    submission
+  } = state.app.submission || {
+    subimission: {
+      id: 1,
```

Comment:
1 here assuming that submissions are given increasing integers as ids. The default is only provided here to ensure default props are passed to the presentational component (of course that component could have its own default props, but I think that adds slightly too much smarts to the presentational component)